### PR TITLE
Tripleme2

### PR DESCRIPTION
**Pull request by TripleMe**
Assignment3: Task06 and Task07 
Assignment2: Updated with datatypes

### CODE REVIEW

**Comment by dachafra** [comment]
6.6 --> falta que UPM sea instancia de university
7.3 --> falta listar las subclases

@ocorcho review assignment2 please

File: Assignment2/TripleMe-MiguelMayoral.ttl [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+@prefix measurement: <http://www.ejercicio2/class01/sensor029/measurement8401#>
+@prefix computer: <http://www.ejercicio2/class01/computer101#>
+@prefix user: <http://www.ejercicio2/class01/computer101/user10A#>
+@prefix xsd <http://www.w3.org/2001/XMLSchema#>
+ 
+:class01 class:includes :sensor029 ;
+	 class:includes :computer101 .
+
+:sensor029 sensor:hasMeasurement :measurement8401 .
+:measurement8401 measurement:hasTemperature : "29"^^xsd:integer ;
```
Comment (ocorcho) [inline Assignment2/TripleMe-MiguelMayoral.ttl]: Wrong here on the usage of : before "29"

File: Assignment2/TripleMe-MiguelMayoral.ttl [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+		 measurement:atTime : "2010-06-12T 12:00:12"^^xsd:dateTime .
```
Comment (ocorcho) [inline Assignment2/TripleMe-MiguelMayoral.ttl]: Wrong here, inside the quotes. Space not needed.

**Comment by ocorcho** [comment]
Check comments

**Pull request closed without merging**